docs/basics/markdown.md

```diff
@@ -26,6 +26,20 @@ span: 2
 
 Catalog provides many specimen types, from images, hints, code snippets to fully interactive HTML and React component specimens. For a full list of all specimens and their features, go to the [Specimen documentation](/specimens)
 
+## Links
+
+Use regular Markdown link to pages within your Catalog documentation.
+
+Each page section can be linked to directly by adding a hash to your link. Click on the link icon next to a heading to jump to that section.
+
+```code|lang-markdown
+- [A link to a page](/specimens)
+- [A link to a page section](/specimens#content)
+- [A link to section on the same page](#example-page)
```

Comment:
I like the idea, it would be kind of like named routes … but it would only shift the automagic into another corner and conflict with other link references, no?

I'm not sure we can solve this issue of changing page paths well if we want to support paths outside of Catalog too. We could also prefix internal paths with `~` or so, but would that be better?